docs/media-buy/api-reference.md

```diff
@@ -1157,31 +1214,57 @@ Completes a human task with resolution. For manual approval tasks, approved oper
 
 Platforms use different status values. AdCP normalizes to:
 - `pending_activation` - Awaiting creative assets
-- `pending_approval` - Under review by ad server
-- `pending_manual` - Awaiting human approval (HITL)
+- `pending_approval` - Under review by ad server (typically minutes/hours)
+- `pending_manual` - Awaiting human approval (HITL - may take days/weeks)
 - `pending_permission` - Blocked by permissions
 - `scheduled` - Future start date
 - `active` - Currently delivering
 - `paused` - Temporarily stopped
 - `completed` - Finished delivering
 - `failed` - Error state
 
-**Important**: Pending states are normal operational states, not errors. Orchestrators must handle them gracefully.
+**Important**: 
+- Pending states are normal operational states, not errors
+- `pending_manual` (HITL) operations have no timeout - they persist until human action
+- Context for HITL operations remains active indefinitely
+- Orchestrators must handle long-running HITL operations gracefully
 
 ### Error Handling
 
-All tools return errors in consistent format:
+All operations return errors using standardized error codes:
+
 ```json
 {
   "error": {
-    "code": "INVALID_PARAMETER",
+    "code": "invalid_parameter",
     "message": "Start date must be in the future",
     "field": "start_date",
-    "suggestion": "Use a date after 2024-02-08"
+    "suggestion": "Use a date after 2024-02-08",
+    "details": {
+      "validation_errors": [
+        {
+          "field": "start_date",
+          "constraint": "future_date",
+          "message": "Must be in the future"
+        }
+      ]
+    }
   }
 }
 ```
 
+**Standard Error Codes:**
+- `invalid_parameter` - Request validation failed
+- `missing_required` - Required field missing
+- `authentication_failed` - Invalid credentials
+- `permission_denied` - Insufficient permissions
+- `not_found` - Resource not found
+- `insufficient_budget` - Budget too low for requirements
+- `inventory_unavailable` - No inventory available
+- `policy_violation` - Content policy violation
+- `internal_error` - Server error
+- `timeout` - Operation timed out
+
 ### Dry Run Mode
 
 Set environment variable `AdCP_DRY_RUN=true` to see platform API calls without execution:
```

Comment:
this seems to be about a server or code, but this is just the spec for the protocol